webpack.config.js

```diff
@@ -92,7 +92,7 @@ config = {
 
     // Add here listener to sccs files?
     demos : {
-        devtool: 'source-map',
+        devtool: 'cheap-eval-source-map',
```

Comment:
faster, nice. I found eval to give me more accurate stack traces but i haven't investigated enough to know exactly why. If this doesn't do what we want, may be worth investigating

Reply:
This works nicely now, I did't know about it, it was @jchen-eb